packages/@vuepress/core/lib/client/components/Content.js

```diff
@@ -10,9 +9,8 @@ export default {
   },
   render (h) {
     const pageKey = this.pageKey || this.$parent.$page.key
-    const pageComponent = getPageAsyncComponent(pageKey)
```

Comment:
This component has been registered in `beforeEnter` hook of vue-router, so there's no need to look it up again, but strangely, the two writings lead to different rendering times for the component.